scripts/context_retrieval/wrangle_llama_index_data.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    def zip_directory(directory_path, output_path):
</pre><p>same goes for this, feels like you might re-use this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/734/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>